libs/sql-schema-describer/src/walkers.rs

```diff
@@ -108,7 +118,7 @@ impl<'a> ColumnWalker<'a> {
 }
 
 /// Traverse a table.
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
We already briefly talked about it, but these debug impls aren't very useful because they will print the whole schema, it is going to be super noisy.

Reply:
Uh, maybe we could make something nicer here?

Reply:
Fixed in https://github.com/prisma/prisma-engines/pull/1372/commits/cae843befd66421d94e6c4f1ebbc4b416ff93ca5